test/08-os-autoinst-obs-auto-submit.t

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env bash
+
+source test/init
+bpan:source bashplus +err +fs +sym
+
+plan tests 6
+
+submit_target_extra="openSUSE:Backports:SLE-15-SP7:Update"
+
+osc_dists_output='
+openSUSE Backports for SLE 15 SP6 openSUSE:Backports:SLE-15-SP6 15.6 standard
+openSUSE Backports for SLE 15 SP5 openSUSE:Backports:SLE-15-SP5 15.5 standard
+openSUSE Backports for SLE 15 SP6:Update  openSUSE:Backports:SLE-15-SP6:Update:GA 1.0 standard 
+openSUSE Backports for SLE 15 SP4 openSUSE:Backports:SLE-15-SP4 15.4 standard'
+
+check_submit_target_extra_expansion() {
+    update_prj="${submit_target_extra##*:}"
+    current_prj="${submit_target_extra%:*}"
+    current_prj="${current_prj%:*}"
+    latest_submit_target_extra_version=$(echo "$osc_dists_output" | grep "$current_prj" | awk -v prj="$current_prj" '{for (i=1; i<=NF; i++) if ($i ~ "^" prj ":") print $i}' | sort --version-sort | tail -n 1)
```

Comment:
It seems terribly wrong that this test duplicates this rather complicated line from the code that's to be tested (even though this is stated in the commit message).